micro-ui/web/micro-ui-internals/packages/modules/workbench/src/configs/UICustomizations.js

```
@@ -809,5 +837,167 @@
       return defaultData
 
     }
-  }
+  },
+  ViewMdmsConfig: {
+    fetchActionItems: (data, props) => {
+      let hostname = window.location.hostname;
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'hostname' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)